tests/Api/ScreenGroupsTest.php

```diff
@@ -163,7 +164,9 @@ public function testGetScreenGroupsScreenRelations(): void
         $this->assertResponseHeaderSame('content-type', 'application/ld+json; charset=utf-8');
         $this->assertJsonContains([
             '@context' => '/contexts/ScreenGroup',
-            '@id' => '/v1/screen-groups',
+            // FIXME: What's the correct `@id` value?
```

Comment:
Remove this and next line. "'@id' => '/v1/screens/'.$ulid.'/screen-groups'," looks correct.

Reply:
Removed in https://github.com/os2display/display-api-service/pull/165/commits/757cdbd81914e0a705b60c61c66c1cc45a4297ee